src/ConnectionManager.h

```diff
@@ -77,7 +75,10 @@ class ConnectionManager {
 #endif
 
 static int debugMessageLevel = ARDUINO_CLOUD_DEBUG_LEVEL;
-inline void debugMessage(char *_msg, uint8_t _debugLevel, bool _timestamp = true, bool _newline = true) {
+inline void debugMessage(char *_msg, int _debugLevel, bool _timestamp = true, bool _newline = true) {
+  if(_debugLevel < 0){
```

Comment:
this check is probably not needed if we only allow messages from 0 up :slightly_smiling_face:

Reply:
mh... better safe than sorry? :)